bodyfetcher.py

```diff
@@ -80,9 +80,6 @@ def make_api_call_for_site(self, site):
             if response["quota_remaining"] - GlobalVars.apiquota >= 1000 and GlobalVars.apiquota >= 0:
                 GlobalVars.charcoal_hq.send_message("API quota rolled over with {} requests remaining.".format(GlobalVars.apiquota))
             GlobalVars.apiquota = response["quota_remaining"]
-        else:
-            GlobalVars.apiquota = 0
-            return
```

Comment:
If the API glitches, don't we *want* to return, so that we don't try to parse bad data? That's what the return is for IIRC.